### DISCUSSION TITLE
Negative values on metrics

### DISCUSSION THREAD

**Discussion by juhotaipale** [laravel/nova-issues#4231] [Q&A] [answered]
Would it be possible to allow negative values on metrics as it allows to show temperature (°C/°F) and signal strength values (dBm)?

**Comment by crynobone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/172966/168718000-036994d5-6703-4c63-8150-309bf825cf9f.png)

I believe it should already be possible?

**Reply by juhotaipale** [reply]
It would be nice to have it formatted with minus f.ex. -11.30 instead of (11.30).
I have temporarily managed this by manually setting the value with ``result(11.3)->prefix('-')`` for now.

**Reply by juhotaipale** [reply]
However, above doesn't work if the values are changing from positive to negative within the displayed range..

**Reply by schwindy** [reply]
+1 on the above.

It feels non-standard to display a negative number in parentheses, unlike anything I have ever seen before 🙏 

It DOES however work with `->format('percent')` :D

Example `calculate`:
```
return $this->sumByHours($request, Example::class, 'example')->format('percent');
```
![image](https://github.com/laravel/nova-issues/assets/11136250/90c81cdb-a96b-414d-800d-3db104e8c651)
